packages/cli/src/util/env/known-error.ts

```diff
@@ -7,7 +9,8 @@ const knownErrorsCodes = new Set([
   'ENV_SHOULD_BE_A_SECRET',
 ]);
 
-export function isKnownError(error: { code?: string }) {
-  const code = error && typeof error.code === 'string' ? error.code : '';
+export function isKnownError(error: unknown) {
+  const code = isErrnoException(error) ? error.code : null;
+  if (!code) return false;
```

Comment:
```suggestion
  const code = isErrnoException(error) ? error.code : '';
```

Reply:
`error.code` can be undefined in the case of `NodeJS.ErrnoException`, so returning early here makes the `code.toUpperCase()` happy below.